libparsec/crates/platform_device_loader/src/native/mod.rs

```diff
@@ -413,3 +482,8 @@ pub async fn save_recovery_device(
 
     Ok(passphrase)
 }
+
+pub fn is_keyring_available() -> bool {
+    // Why tmp ? Because macos doesn't allow empty user
```

Comment:
Actually, it's not macOS, it's a `keyring-rs` choice, because it seems that macOS treats "empty" as a wildcard. Maybe we can even include a link to the issue:
```suggestion
    // Using "tmp" as user, because keyring-rs forbids the use of empty string due to an issue in macOS.
    // See: https://github.com/hwchen/keyring-rs/pull/87
```